cider/tests/nextgen/jitlib/JITLibTest.cpp

```diff
@@ -143,6 +146,43 @@ TEST_F(JITLibTests, LogicalOpTest) {
       false, true, true, [](JITValue& a, JITValue& b) { return !a; });
 }
 
+TEST_F(JITLibTests, TupleTest) {
+  LLVMJITModule module("Test Module");
+  JITTuple layer1(JITTypeTag::TUPLE, JITBackendTag::LLVMJIT);
+  JITTuple layer2(JITTypeTag::TUPLE, JITBackendTag::LLVMJIT);
+  JITTuple layer3(JITTypeTag::TUPLE, JITBackendTag::LLVMJIT);
+
+  JITFunctionPointer function1 = module.createJITFunction(JITFunctionDescriptor{
```

Comment:
I can't see the tuple is used in either argument type or return type

Reply:
discuss offline